string image_path
---
string fruit_type
